docs/topics/spiders.rst

```diff
@@ -259,6 +259,29 @@ to give data more structure you can use :class:`~scrapy.Item` objects::
             for href in response.xpath('//a/@href').getall():
                 yield scrapy.Request(response.urljoin(href), self.parse)
 
+.. _spiderargs:
+
+
+    .. method:: update_settings()
+        
+        The ``update_settings`` method allows you to modify the settings of a spider. It is a classmethod that is called to 
+        override the configuration of your spider. When called, it receives the spider's settings and passes ``custom_settings``
+        or an empty dictionary to the ``setdict`` method to register the setting with a priority of spider. It is recommended to
+        use ``update_settings`` rather than ``custom_settings`` when creating class hierarchies for spiders. This is because the parent's
+        ``custom_settings`` may be shadowed, but the parent's ``update_settings`` method can still be called.
+
+here is an example::
```

Comment:
This should have the same indentation as the paragraph above.

Reply:
I've indented the code example to the same indentation as the paragraph